Fixed #35748 -- Documented that Field.formfield() may return None. (#18570)
django/django: docs/ref/models/fields.txt

    .. method:: formfield(form_class=None, choices_form_class=None, **kwargs)

        Returns the default :class:`django.forms.Field` of this field for

>         :class:`~django.forms.ModelForm`, or ``None`` to exclude the field from
>         the form.

Reviewer: ```suggestion
        :class:`~django.forms.ModelForm`. If ``None`` is returned, this field
        is excluded from the :class:`~django.forms.ModelForm`.
```

>         :class:`~django.forms.ModelForm`. If ``None`` is returned, this field
>         is excluded from the :class:`~django.forms.ModelForm`.

Reviewer: ```suggestion
        :class:`~django.forms.ModelForm`.

        If :meth:`~Field.formfield` is overridden to return ``None``, this field
        is excluded from the :class:`~django.forms.ModelForm`.
```
`None` will not be returned by the default implementation (which you can see when clicking the "source" link), so I think it's worth us emphasizing that it's been overridden
